convert/snippet_mutator.rb

```diff
@@ -0,0 +1,74 @@
+$LOAD_PATH.unshift File.dirname(__FILE__)
+
+require 'active_support/all'
+require 'byebug'
+require 'section_mutator.rb'
+
+module SnippetMutator
+
+  class Csharp
+    REPLACE = {
+      /&/                                                => '',
+      /->/                                               => '.',
+      /kuzzleio::/                                       => '',
+      /Kuzzleio::/                                       => '',
+      /std::string/                                      => 'string',
+      /query_options options;/                           => 'QueryOptions options = new QueryOptions();',
+      /SearchResult\*/                                   => 'SearchResult',
+      /validation_response \*validation_response/         => 'validation_response validation_response',
+      /e.what\(\)/                                       => 'e.getMessage()'
```

Comment:
Too java centric (even so, in Java we usually use toString() to print exceptions, as done in our code snippets)

Reply:
I'm waiting a proper getter in c#. I think in c# public method have to start with an uppercase character like in Go so a lot of things is gonna change

Reply:
Ok, I will change the convert script.
C# exceptions inherits from `System.ApplicationException`